codex-rs/rmcp-client/src/oauth.rs

```diff
@@ -83,15 +103,15 @@ impl fmt::Display for CredentialStoreError {
 
 impl std::error::Error for CredentialStoreError {}
 
-trait CredentialStore {
+trait KeyringStore {
     fn load(&self, service: &str, account: &str) -> Result<Option<String>, CredentialStoreError>;
     fn save(&self, service: &str, account: &str, value: &str) -> Result<(), CredentialStoreError>;
     fn delete(&self, service: &str, account: &str) -> Result<bool, CredentialStoreError>;
 }
 
-struct KeyringCredentialStore;
+struct RealKeyringStore;
 
-impl CredentialStore for KeyringCredentialStore {
+impl KeyringStore for RealKeyringStore {
```

Comment:
Why are the other ones "not real"? Add a docstring? Or maybe rename to `DefaultKeyringStore`?

Reply:
The other one is a mock